jenkins/docker/docker-copy.jenkinsfile

```diff
@@ -45,7 +50,7 @@ pipeline {
                 stage('Parameters Check') {
                     steps {
                         script {
-                            currentBuild.description = "${SOURCE_IMAGE_REGISTRY}/${SOURCE_IMAGE} to ${DESTINATION_IMAGE_REGISTRY}/${DESTINATION_IMAGE}"
+                            currentBuild.description = """ALL_TAGS: ${ALL_TAGS}<br>Copying: ${SOURCE_IMAGE_REGISTRY}/${SOURCE_IMAGE} to ${DESTINATION_IMAGE_REGISTRY}/${DESTINATION_IMAGE}"""
```

Comment:
""" is intentional or by mistake? @peterzhuamazon

Reply:
It is intentional to parse html tags.